.github/workflows/build_and_test.yml

```diff
@@ -140,11 +145,10 @@ jobs:
   build:
     name: "Build modules (${{ format('{0}, {1} job', needs.configure-jobs.outputs.branch, needs.configure-jobs.outputs.type) }}): ${{ matrix.modules }} ${{ matrix.comment }} (JDK ${{ matrix.java }}, ${{ matrix.hadoop }}, ${{ matrix.hive }})"
     needs: [configure-jobs, precondition]
-    # Run scheduled jobs for Apache Spark only
-    # Run regular jobs for commit in both Apache Spark and forked repository
+    # Run for all scheduled jobs
+    # Run for all other jobs only when changes exist
```

Comment:
I think the original note still valid in here, and more useful. WDYT?

Reply:
I have reverted and amended the original comment.